userguide/recommender/custom-model-ios.md

```diff
@@ -0,0 +1,50 @@
+# Custom Recommender Model for iOS Applications
+
+The Turi Create Recommender Model is also available for use in your iOS/macOS
+Applications as a custom model via exporting to Core ML. After creating your model in
+Turi Create, you need to export your model to Core ML by leveraging the simple 
+`export_coreml` API as follows:
+
+```python
+# assume my_recommender is the trained Turi Create Recommender Model
+my_recommender.export_coreml("MyRecommender.mlmodel")
+```
+
+After you drag and drop the exported Core ML model in your iOS Application, it will  look something like this:
+![Core ML Model Screenshot in Xcode](xcode-shot.png)
+
+As you can see above, we need to give the model two inputs, `interactions` and `k`. 
```

Comment:
You are missing the instructions to include the binary into the app and where to get it from and all the instructions on how to link the binary to your app (i.e adding the dylib into a new phase etc.)